config/agent/hirte-default.conf

```diff
@@ -0,0 +1,31 @@
+# This is the default configuration values for hirte-agent
+
+[hirte-agent]
+#
+# The unique name of this agent.
+# It's mandatory to set this option for each hirte-agent.
+NODE_NAME=
```

Comment:
Caps, should be NodeName

Reply:
Sorry, forgot to change the default file added in the last commit, too many refactorings :-(
Fixed